README.md

```diff
@@ -225,6 +266,7 @@ Sending data to Datadog US (default) and configuring a few checks.
     - { role: Datadog.datadog, become: yes }
   vars:
     datadog_api_key: "123456"
+    datadog_agent_major_version: 6
```

Comment:
The main Playbook example should install the latest version. Let's add another example to showcase `datadog_agent_major_version` and maybe `datadog_agent_version`.